R/timeseriessummary-data.R

```diff
@@ -413,8 +413,14 @@ unNestCorrectionParameters <- function(corrections, timezone) {
   deviationType <- ".dplyr"
   windowSizeInMinutes <- ".dplyr"
   ungroup <- ".dplyr"
+  
+  if (corrections[["type"]]=="CopyPaste" || corrections[["type"]]=="Freehand" || corrections[["type"]]=="DeleteRegion" || corrections["type"]=="RevertToRaw") {
+    corrections[["parameters.dummy"]] <- ""
```

Comment:
@lindsaycarr how bad of a taste does this leave in your mouth? the problem is there aren't any parameters.X fields in these types of corrections